classes/importer.php

```diff
@@ -700,44 +676,55 @@ protected function _getupdatesfromldap($ldap, $ldaptimestamp = null) {
      * @see auth_plugin_ldap::sync_users()
      * @throws Exception on database/SQL related failures
      */
-    protected function _updatemoodleaccounts(array $data) {
+    protected function updatemoodleaccounts(array $data) {
         global $CFG, $DB;
         if (!count($data)) {
             return;
         }
-        // Convert key names to lower case in each record
-        // (Should already be lower case.
-        // $dataLS = array();
-        // foreach ($data as $record) {
-        // $recordLS = array();
-        // foreach ($record as $key => $value) {
-        // $recordLS[core_text::strtolower($key)] = $value;
-        // }
-        // $dataLS[] = $recordLS;
-        // }
         $usertblname = $CFG->prefix . 'user';
         $stagingtblname = $CFG->prefix . self::MOODLE_TEMP_TABLE;
-        // attempt to delete a previous temp table
+        // Attempt to delete a previous temp table.
         $deletetemptablesql = "DROP TEMPORARY TABLE IF EXISTS {$stagingtblname}";
         $createtemptablesql = <<< EOL
 CREATE TEMPORARY TABLE {$stagingtblname}
```

Comment:
why not use `database_manager::create_temp_table()` here instead?

for example, see https://github.com/moodle/moodle/blob/MOODLE_401_STABLE/mod/forum/report/summary/classes/summary_table.php#L825